src/Nri/Ui/Table/V8.elm

```diff
@@ -117,27 +119,86 @@ rowHeader options =
 
 
 
+-- Model
+
+
+type alias Config =
+    { css : List Style, alternatingRowColors : Bool }
+
+
+{-| Attribute to configure the table
+-}
+type Attr
```

Comment:
🎨 We call this `Attribute` everywhere, can we change it here for consistency?

<img width="241" alt="Screenshot 2024-11-06 at 12 14 29 PM" src="https://github.com/user-attachments/assets/57d56e4c-01fb-4ea2-be18-8175e9aa1b65">